bgcval2/timeseries/timeseriesTools.py

```diff
@@ -43,6 +43,18 @@
 """
 
 
+def choose_best_var(nc, keys):
+    """
+    Takes the list of keys and chooses the first one that exists in the input file.
+    Useful if fields change for no reason.
+    """
+    for key in keys:
+        if key not in nc.variables.keys():
+            continue
+        return nc.variables[key]
+    raise KeyError(f'choose_best_var: unable to find {keys} in {nc.filename}')
```

Comment:
can you not import the one above? It's identical

Reply:
and you know this is not quite the *best* var but rather, the last var in the keys list that's present in nc.variables - is this the behaviour you're after?

Reply:
It should be the first one that overlaps.